src/logic/cleanBuildVersion.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import { Platform } from "react-native";
+
+const mega = 1048576;
+export const getAndroidArchitecture = (buildVersion?: ?string) => {
+  const buildVersionNumber = parseInt(buildVersion, 10);
+  if (!buildVersionNumber) return "";
+
+  if (Platform.OS === "android" && buildVersionNumber) {
+    const platforms = ["armeabi-v7a", "x86", "arm64-v8a", "x86_64"];
+    return platforms[Math.floor(buildVersionNumber / mega) - 1];
```

Comment:
are we sure about the `- 1`?

Reply:
also could you add a comment with a link to the sourcecode that do this calculation? it's somewhere in a build gradle script